shared/src/commonMain/kotlin/org/hyperskill/app/topics_repetitions/presentation/TopicsRepetitionsReducer.kt

```diff
@@ -41,7 +41,13 @@ class TopicsRepetitionsReducer : StateReducer<State, Message, Action> {
                 if (state is State.Content && state.hasNextTopicsToLoad) {
                     state.copy(isLoadingNextTopics = true) to setOf(
                         Action.FetchNextTopics(
-                            nextPage = (state.topicsRepetitions.count() / TopicsRepetitionsActionDispatcher.TOPICS_PAGINATION_SIZE) + 1
+                            nextPage = if (
+                                state.topicsRepetitions.size % TopicsRepetitionsActionDispatcher.TOPICS_PAGINATION_SIZE == 0
```

Comment:
I guess it's better to move `TOPICS_PAGINATION_SIZE` from dispatcher to feature. We should not refer dispatcher from reducer and vice versa.